extensions/amp-story/1.0/amp-story.js

```diff
@@ -2467,6 +2470,30 @@ export class AmpStory extends AMP.BaseElement {
       extensionsFor.installExtensionForDoc(ampdoc, 'amp-analytics');
     }
   }
+
+  /**
+   * If there are inline localization strings, register as current document language.
+   */
+  registerInlineLocalizationStrings_() {
+    const inlineStringsEl = this.win.document.querySelector(
+      'script[amp-localization="amp-story"]'
+    );
+    if (
+      !inlineStringsEl ||
+      inlineStringsEl.getAttribute('i-amphtml-version') !=
+        getMode(this.win).rtvVersion
```

Comment:
I'd really like us to stick with strict equalities

```suggestion
      inlineStringsEl?.getAttribute('i-amphtml-version') !==
        getMode(this.win).rtvVersion
```